src/nvim/terminal.c

```diff
@@ -217,10 +214,14 @@ Terminal *terminal_open(TerminalOptions opts)
   vterm_set_utf8(rv->vt, 1);
   // Setup state
   VTermState *state = vterm_obtain_state(rv->vt);
+  // Only relevant in terminal true color mode
+  if (true_color) {
+    vterm_state_set_bold_highbright(state, true);
```

Comment:
No, if you are talking about the `g:terminal_color` options then those were only used in true color mode. They were never used to change the color index. They only changed the rgb (true color value) of the cell.

Anyways onto the highbright stuff. The reason I have it as it is is because:

1. In non true color more you already have it off because you want to respect the user terminal setting regarding highbright. I can see the having an option for it only if you have highbright off in your normal terminal and want to have it on in neovim terminal.

However, this setting will only work one way. That is, if the user terminal has highbright on, you can't turn it off in neovim. So this option could be kinda confusing in non true color mode.

2. In true color mode this should probably be on by default. As I stated previously, it seems like most if not all linux terminals defaults to highbright on. So I would say that the expected behaviour of the neovim terminal would be to have it on.

This will clash with the default to off in non true color mode.

So I think that the only sane way to do this is to have the option only when in true color mode. It's also a bit more logical because then all the `g:terminal_` options are only used in true color mode.